listenbrainz/db/lb_radio_artist.py

```diff
@@ -0,0 +1,151 @@
+from collections import defaultdict
+import json
+from random import randint
+from typing import List
+import uuid
+
+from flask import current_app
+import psycopg2
+from psycopg2.extras import DictCursor
+
+from listenbrainz.webserver import ts_conn
+
+
+def lb_radio_artist(mode: str, seed_artist: str, max_similar_artists: int, num_recordings_per_artist: int, pop_begin: float,
+                    pop_end: float) -> List[dict]:
+    """
+        Fetch recordings for LB Radio's similar artists element.
+
+        Given a seed artist mbid, find similar artists given other parameters and then return
+        a dict of artist_mbids that contain lists of dict as such:
+            {
+              "recording_mbid": "401c1a5d-56e7-434d-b07e-a14d4e7eb83c",
+              "similar_artist_mbid": "cb67438a-7f50-4f2b-a6f1-2bb2729fd538",
+              "total_listen_count": 232361
+            }
+
+        Troi will take this data and complete processing it into a complete playlist.
+
+        parameters:
+
+        mode: LB radio mode, must be one of: easy, medium, hard.
+        seed_artist: artist mbid of the seed artist for similar artists
+        num_recordings_per_artist: Return up to this many recordings for each artist.
+        pop_begin: Popularity range percentage lower bound. A popularity range is given to narrow down
+                   the recordings into a smaller target group. The most popular track(s) on
+                   LB have a pop percent of 100. The least popular tracks have a score of 0.
+        pop_end: Popularity range percentage upper bound. See above.
+    """
+
+    query = """WITH mbids(mbid, score) AS (
```

Comment:
`score` can be removed from here (also from the parameters being passed to this query).